src/App.tsx

```diff
@@ -1,18 +1,30 @@
 import "./App.css";
 
-import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
-import { routes } from "./presentation/common/Routes";
+import { BrowserRouter as Router, Switch } from "react-router-dom";
+
+import {
+  PrivateRoute,
+  PublicRoute,
+  routes,
+} from "./presentation/common/Routes";
 
 // Component imports
 import Home from "./presentation/Home";
+import Login from "./presentation/Login";
+import Initial from "./presentation/Initial";
+import Auth0 from "./presentation/Auth0";
```

Comment:
Should we make it to one line import? Like,

```
import { Home, Login, Initial } from "...";
```

Reply:
done